cassandra/cqlengine/columns.py

```diff
@@ -16,6 +16,8 @@
 from datetime import date, datetime
 import logging
 import six
+import struct
+import iso8601
```

Comment:
I know the OP requested this, but I'm luke warm on adding a new dep for date parsing. We usually leave things like this to the application layer. The main reason is that we want a lean dependency list because of the cqlsh integration (Cassandra has to carry along tarballs of all dependencies). This is not stated anywhere, so maybe should I should add some discussion to the contribution doc.

All of the above said, the main constraint does not apply to the mapper layer, so we could actually get away with it here. I just want to point it out to make sure it's warranted, and make sure we don't leak it into the core. We also have to consider then that we are saying string dates are ok for the mapping layer but not for core.